### DISCUSSION TITLE
Formalize Proposals Process

### DISCUSSION THREAD

**Discussion by electronjoe** [magma/magma#5366] [Ideas]
## Summary

This Discussion post is intended to elevate visibility of the GH Issue and associated Design Doc - containing a Proposal for a formal Proposal Process at Magma.

* [GH Issue](https://github.com/magma/magma/issues/4888)
* [Draft Design Doc](https://github.com/magma/magma/pull/4907) - to be discussed on GH Issue

## Background

As an open source project, it is important that all contributors are able to observe substantitive Proposals, comment on them, and be able to discover historical discussions. This preserves important historical context for decisions, reduces load on repeat Proposals, and enables the broad community to suggest and execute large scale changes.

It is important that this Proposal process be appropriately light-weight, clearly documented, and exeucted within a reasonable and advertised timeline.

Proposals may take all sorts of shapes, including:

* Design Proposals
* Repository change Proposals
* Process Proposals

**Comment by hcgatewood** [comment]
I'm very much in favor of GitHub Discussions playing a large role in the finalized process. It's been a great experience so far

**Reply by electronjoe** [reply]
I've updated the GH Issue with some of my thoughts as I try to nail this down (how we would use GH Discussion).  We can discuss today: #4888
